scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1882,116 +1882,6 @@
             (is (= (:slave-id offer-1) (:slave-id task-1)))
             (is (= (:slave-id offer-2) (:slave-id task-2)))))))))
 
-(deftest test-in-order-status-update-processing
```

Comment:
Why delete?

Reply:
Moved it to another file, which I forgot to `git add` 😳